assets/performanceprofile/configs/99-workload-partitioning.conf

```diff
@@ -0,0 +1,6 @@
+{{- if . }}
+[crio.runtime.workloads.{{ .Workload }}]
+activation_annotation = "target.workload.openshift.io/{{.Workload}}"
+annotation_prefix = "resources.workload.openshift.io"
+resources = { "cpushares" = 0, "cpuset" = "{{ .ReservedCpus}}" }
```

Comment:
The not adding of cpushares is not self documenting here. Not sure the best place to say it, but adding a blurb somewhere like
```
cpushares must be set to zero here. Not setting it causes the TOML to not be parsed by CRI-O. Further, it should not be configurable through the API, as the Kubelet will inject the correct cpu share annotations according to the pod spec, rom carried patches in https://github.com/openshift/kubernetes/pull/706
```
or something like that. Can you think of a good place to put such a note @eggfoobar

Reply:
I mention both of those because I forgot both of them and had to refresh my memory :)

Reply:
Great idea, I added comments to the method that generates this files, as well as a golang template comment to help out any future developers that look at this file.